web/styles/left_sidebar.css

```diff
@@ -1365,29 +1365,23 @@ li.topic-list-item {
     }
 
     #hide-more-direct-messages {
-        display: block;
+        display: grid;
```

Comment:
This isn't the right approach here. There is already a grid definition for the left sidebar rows; `#hide-more-direct-messages` should be added to the list of selectors that share the grid definition. See #30632 for a recent example of what the approach will look like here (roughly).

Reply:
I've pushed the fix for this in the first commit. `#hide-more-direct-messages` shares the grid definition, but still inherits a few `grid-` properties.